### PR TITLE
Automated PR for SCEA

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/SCEA.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+             would be the Licensor.</p>
+        </li>
+        <li>
+          <b>5.</b>
+          <p>Redistribution in Source Form</p>
+          <p>You may redistribute copies of the Software, modifications or derivatives thereof in Source Code
+             Form, provided that You:</p>
+        </li>
+      </list>
+      <list>
```
Comment (sam-ellis) [inline src/SCEA.xml]: This list should be merged into list 5) above.

File: src/SCEA.xml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        </li>
+        <li>
+          <b>6.</b>
+          <p>Redistribution in Object Form</p>
+          <p>If You redistribute copies of the Software, modifications or derivatives thereof in Object Form
+             only (as incorporated into finished goods, i.e. end user applications) then You will not have
+             a duty to include any copies of the code, this License, copyright notices, other attributions
+             or documentation.</p>
+        </li>
+        <li>
```
Comment (sam-ellis) [inline src/SCEA.xml]: Bullet 6) is part of the previous list not list a), b)... Needs to be outdented,

File: src/SCEA.xml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+             jurisdiction and venue of such courts for purposes of any such action. In addition, each party
+             hereby waives the right to a jury trial in any action or proceeding related to this
+             Agreement.</p>
+        </li>
+        <li>
+          <b>10.</b>
+          <p>Copyright Notice for Redistribution of Source Code</p>
+        </li>
+      </list>
+      <p>Copyright 2005 Sony Computer Entertainment Inc.</p>
```
Comment (sam-ellis) [inline src/SCEA.xml]: Change to \<copyright> line?

Comment (jlovejoy) [inline src/SCEA.xml]: this is all part of a header, so added that

File: src/SCEA.xml (hunk 2 above)
Comment (jlovejoy) [inline src/SCEA.xml]: fixed

**Pull request merged**